src/pkgcheck/checks/metadata.py

```diff
@@ -1358,18 +1358,20 @@ class DescriptionCheck(Check):
     """DESCRIPTION checks.
 
     Check on length (<=80), too short (<10), or generic (lifted from eclass or
-    just using the package's name).
+    just using the package's name), or ending with a full stop.
     """
 
     known_results = frozenset([BadDescription])
 
     def feed(self, pkg):
-        desc = pkg.description
+        desc: str = pkg.description
         s = desc.lower()
         if s.startswith("based on") and "eclass" in s:
             yield BadDescription("generic eclass defined description", pkg_desc=desc, pkg=pkg)
         elif s in (pkg.package.lower(), pkg.key.lower()):
             yield BadDescription("generic package description", pkg_desc=desc, pkg=pkg)
+        elif desc.endswith(tuple(".,:;")):
```

Comment:
Could I get explanation of that new rule? I'm unable to understand it.

Reply:
@soapgentoo

Reply:
Long time ago this was considered for repoman... and, if I don't misremember, it was finally reverted because of people thinking differently on this :/
https://bugs.gentoo.org/438976